samples/apply-hillshade-renderer-to-raster/src/main/java/com/esri/arcgismaps/sample/applyhillshaderenderertoraster/screens/ApplyHillshadeRendererToRasterScreen.kt

```diff
@@ -0,0 +1,259 @@
+                        modifier = Modifier.wrapContentHeight(),
+                        sheetState = controlsBottomSheetState,
+                        onDismissRequest = { isBottomSheetVisible = false }
+                    ) {
+                        HillshadeRendererOptions(
+                            onApplyRendererClicked = mapViewModel::updateRenderer,
+                            onDismiss = { isBottomSheetVisible = false }
+                        )
+                    }
+                }
+            }
+        },
+        floatingActionButton = {
+            if (!isBottomSheetVisible) {
+                FloatingActionButton(
+                    modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { isBottomSheetVisible = true }
+                ) { Icon(Icons.Filled.Settings, contentDescription = "Hillshade renderer options") }
+            }
+        }
+    )
+}
+
+@Composable
+fun HillshadeRendererOptions(
+    onDismiss: () -> Unit,
+    onApplyRendererClicked: (Double, Double, SlopeType) -> Unit
+) {
+    var altitude by remember { mutableIntStateOf(0) }
+    var azimuth by remember { mutableIntStateOf(0) }
+    var selectedSlopeType by remember { mutableStateOf<SlopeType>(SlopeType.None) }
+    val slopeTypes = listOf(
+        SlopeType.None, SlopeType.Degree, SlopeType.PercentRise, SlopeType.Scaled
+    )
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 12.dp, horizontal = 8.dp),
+        verticalArrangement = Arrangement.spacedBy(12.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            OutlinedButton(onClick = onDismiss) {
+                Text("Cancel")
+            }
+            Text(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 12.dp)
+                    .weight(1f),
+                text = "Hillshade Renderer Settings",
+                style = MaterialTheme.typography.labelLarge,
+                textAlign = TextAlign.Center
+            )
+            Button(onClick = {
+                onApplyRendererClicked(
+                    altitude.toDouble(),
+                    azimuth.toDouble(),
+                    selectedSlopeType
+                )
+            }) {
+                Text("Apply")
+            }
+        }
+        HorizontalDivider()
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(text = "Altitude:", style = MaterialTheme.typography.labelLarge)
+            Text(text = altitude.toString(), style = MaterialTheme.typography.labelLarge)
+        }
+        // Altitude Slider
+        Slider(
```

Comment:
A `Slider` might not be the most appropriate composable for this.
[According to Material 3 guidelines](https://m3.material.io/components/sliders/guidelines#5816d286-70df-4eb4-88bf-0a4786a8e2a0)
> Changes made with sliders must take effect immediately, so people can understand the effects of their selection as they're moving the slider.

and as you have it, the user has to click on "apply" to see the changes. 

I would suggest calling `updateRenderer` with the new values when the user changes the values in the slider. Then the user would see the changes right away.